my-react-app/src/redux/game/actions.js

```diff
@@ -0,0 +1,4 @@
+export const actions = {
+  handleClick: 'HANDLE_CLICK',
+  jumpTo: 'JUMP_TO'
+};
```

Comment:
Create an object called `actionsCreators` here. That object would contain the functions that create the actual actions. Ask me if you need help with that .